tests/unit/browser/test_tab.py

```diff
@@ -91,6 +91,8 @@ def __init__(self, win_id, mode_manager, parent=None):
     def _install_event_filter(self):
         pass
 
+    # pylint: enable=abstract-method
+
```

Comment:
No need for this, as pylint already only turns things off for this class and it's needed for the entire class.